draft-ietf-quic-manageability.md

```diff
@@ -287,9 +287,14 @@ Hello, and the server can send 1-RTT data as soon as it has sent its Server
 Hello. The Client Completion flight contains at least one Handshake packet and
 could also include an Initial packet. QUIC packets in separate contexts during
 the handshake can be coalesced (see {{coalesced}}) in order to reduce the
-number of UDP datagrams sent during the handshake.  QUIC packets can be lost
-and reordered, so packets within a flight might not be sent close in time,
-though the sequence of the flights will not change, because one flight depends
+number of UDP datagrams sent during the handshake.
+
+Handshake packets can arrive out-of-order without impacting the handshake as
+long as the reordering did not cause extensive delays that trigger a
+spurious Probe Timeout ({Section 6.2 of RFC9002}).
+If QUIC packets get lost or reordered, packets belonging
+to the same flight might not be observed in close succession in time, though
```

Comment:
s/in close succession in time/in close time succession/

Reply:
```suggestion
to the same flight might not be observed in close time succession, though
```